pyproject.toml

```diff
@@ -33,21 +33,18 @@ google-cloud-storage = ">=1.24.1 <=1.28.1"
 jupyter = "^1.0.0"
 kornia = "^0.1.4"
 numpy = ">=1.17 < 1.18"
-nuscenes-devkit = ">=1.0.2 <1.0.3"
 plotly = "^4.4.1"
 pyquaternion = "^0.9.5"
-pytorch-ignite = "^0.3.0"
-torch = "~1.4.0"
-torchvision = "~0.5"
 yacs = "^0.1.6"
 pycocotools = "^2.0.0"
 codetiming = "^1.2.0"
 pandas = "^1.0.1"
 tqdm = "^4.45.0"
 dask = {extras = ["complete"], version = "^2.14.0"}
 dash = "1.12.0"
-tensorflow = "^2.2.0"
 click = "^7.1.2"
+scikit-learn = "^0.23.2"
```

Comment:
Why are these two requirements added?

Reply:
- Removed scikit-learn: as it was used for train-split but we don't need as we don't model training code.
- We need opencv-python, as it is used in bounding box 2d and 3d and its been used in perception stats notebook.